source/index.html.md

```diff
@@ -380,21 +395,55 @@ Example response (snippet):
     "status": {
         "code": 20000
     },
-    "processed_at": "2018-07-09T12:58:56+00:00",
-    "recurring": true,
-    "_embedded": { "card": { "id": "58dabba0-e9ea-4133-8c38-bfa1028c1ed2" } }
+    "processed_at": "2019-08-14T12:58:56+00:00",
+    "credential_on_file": {
+        "type": "recurring"
+    },
+    "_embedded": {
+        "self": { "href": "/authorizations/04dce6bf-ca17-42c5-a61e-5fdb4b87310a" }
+    }
 }
 ````
 
 This should be followed by a capture except when the amount is `0`.
 
-Subsequent authorizations are made similarly, but neither CSC nor PARes (see
+Subsequent authorizations are made by using `credential_on_file[previous]` which
+points to the previous authorization in the series; for `installment`,
+`recurring`, and `unscheduled`, where subsequent transactions are
+merchant-initiated, neither CSC nor 3-D Secure values (see
 [3-D Secure](#3-d-secure)) would be included.
 
-An initial recurring authorization can also be made using the `applepay` and
-`mobilepayonline` payment methods; subsequent recurring payments, however, must
-be made using the `card` payment method using the card details of the initial
-recurring authorization.
+````shell
+curl -X POST \
+  https://gateway.test.clearhaus.com/authorizations \
+  -u <your-api-key>:  \
+  -d "amount=2050"    \
+  -d "currency=EUR"   \
+  -d "credential_on_file[previous]=/authorizations/04dce6bf-ca17-42c5-a61e-5fdb4b87310a" \
+  -d "card[pan]=4111111111111111" \
+  -d "card[expire_month]=06"      \
+  -d "card[expire_year]=2022"
+````
+
+Example response (snippet):
+
+````json
+{
+    "id": "e3e9d215-6efc-4c0e-b3d7-2226057c6de8",
+    "status": {
+        "code": 20000
+    },
+    "processed_at": "2019-08-14T12:58:56+00:00",
+    "credential_on_file": {
+        "type": "recurring"
+    }
+}
+````
+
+First in series transactions can also be made using the `applepay` and
+`mobilepayonline` payment methods; subsequent in series merchant-initiated
```

Comment:
I would prefer it
> subsequent in series _of_ merchant initiated
 
(no dash)

Reply:
Except from the framework name, card schemes seem to consistently use the version with a hyphen.

What about
> however, subsequent merchant-initiated transactions must (...)

?